packages/cli/__tests__/zosfiles/__unit__/compare/ds/Dataset.handler.unit.test.ts

```diff
@@ -108,6 +110,45 @@ describe("Compare data set handler", () => {
             expect(logMessage).toEqual("compared string");
             expect(DiffUtils.getDiffString).toHaveBeenCalledTimes(1);
         });
+        it("should compare two data sets in terminal with --context-lines option", async () => {
+            const contextLinesArg: number = 2;
+            const processArgCopy: any = {
+                ...processArguments,
+                arguments:{
+                    ...processArguments.arguments,
+                    contextLines: contextLinesArg
+                }
+            };
+            const options: IDiffOptions = {
+                contextLinesArg,
+                outputFormat: "terminal"
+            };
+
+            DiffUtils.getDiffString = jest.fn(async () => {
+                return "compared string";
+            });
```

Comment:
I know that subsequent tests are not looking for any output of the called functions, however please keep in mind that this stile of mocking may result in unexpected test failues.
For example, someone adding tests (after this one) might expect the real behavior of the `getDiffString` method and they will get a mocked version instead.
Suggestion: For future tests, please try using `jest.spyOn` (if possible) or `jest.mock` when dealing with completed modules 😋

Reply:
I've added this mocked method approach